accounts/swagger/connection.py

```diff
@@ -0,0 +1,17 @@
+from drf_yasg2 import openapi
+from drf_yasg2.utils import swagger_auto_schema
+from accounts.views.connection_views import create_connection
+
+create_connection_endpoint = swagger_auto_schema(
+    method= 'Post',
+    request_body= openapi.Schema(type=openapi.TYPE_OBJECT, required= ['name', 'relation'], 
```

Comment:
- remove `name` and `relation` from required fields as they are not required.
- add `image` to the required fields in addition to adding the as input in `properties`